doc/references/ImplementationOfVHDL.rst

```diff
@@ -72,7 +72,7 @@ The latest version is 2008. Many features have been added, and GHDL
 doesn't implement all of them.
 
 You can select the VHDL standard expected by GHDL with the
-``--std=VER`` option, where ``VER`` is one of the left column of the
+``--std=<STANDARD>`` option, where ``<STANDARD>`` is one of the left column of the
```

Comment:
The conent below is a list, not a table. This can be misleading. Despite not being something introduced by you, would you mind fixing it? Just "*... is one of the left column of <strike> the table below</strike>:*".